unit-tests/test-fw-updater.py

```diff
@@ -0,0 +1,141 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2021 Intel Corporation. All Rights Reserved.
+
+#test:priority 0
+#test:device L500*
+#test:device D400*
+
+import pyrealsense2 as rs, sys, os, subprocess, re
+from rspy import devices, log, test
+
+def filesin( root ):
+    # Yield all files found in root, using relative names ('root/a' would be yielded as 'a')
+    for (path,subdirs,leafs) in os.walk( root ):
+        for leaf in leafs:
+            # We have to stick to Unix conventions because CMake on Windows is fubar...
+            yield os.path.relpath( path + '/' + leaf, root ).replace( '\\', '/' )
+
+def find( dir, mask ):
+    pattern = re.compile( mask )
+    for leaf in filesin( dir ):
+        if pattern.search( leaf ):
+            #log.d(leaf)
+            yield leaf
+
+def get_bundled_fw_version( product_line ):
```

Comment:
Same with this:
Can we create a `repo` module that would automatically look for the LRS repo root and then would have a function to get the FW version for a product line?